src/display.ts

```diff
@@ -98,11 +98,30 @@ type FeatureInfo = {
 };
 
 const drawFeature = (svg: SVGSVGElement, face: Face, info: FeatureInfo) => {
-  const feature = face[info.name];
+  const feature = Object.assign({}, face[info.name]);
   if (!feature || !svgs[info.name]) {
     return;
   }
 
+  if (
+    face.aging &&
+    face.aging.enabled &&
+    info.name === "hair" &&
+    feature.id === "short-bald" &&
```

Comment:
This handles one hair style, but not any of the others. Maybe in generate.ts, short-bald and bald should never be selected, and then here they can be activated at some age cutoff for other hair styles.

Fun way of doing that and having it feel random - somehow pick a deterministic random number seeded with some of the info in the face object. Use that to determine the age balding starts (short-bald) and ends (bald).